src/simditor-mention.coffee

```diff
@@ -224,17 +230,33 @@ class SimditorMention extends SimpleModule
 
       $itemEl.appendTo($itemsEl).data 'item',item
 
-    @popoverEl.on 'mouseenter', '.item', (e)->
-      $(@).addClass 'selected'
-        .siblings '.item'
-        .removeClass 'selected'
-    @popoverEl.on 'mousedown touchend','.item', (e)=>
+    @popoverEl.on 'mouseenter', '.item', (e)=>
+      @updateSelectedItem($(e.currentTarget))
+
+    @popoverEl.on 'mousedown','.item', (e)=>
       @selectItem()
       false
 
-    $itemsEl.on 'mousewheel', (e,delta)->
-      $(@).scrollTop $(@).scrollTop() - 10*delta
-      false
+    @popoverEl.on 'touchstart', '.item', (e)=>
+      if !@popoverIsTouching
+        @updateSelectedItem($(e.currentTarget))
+        touches = e.originalEvent.targetTouches[0]
+        @popoverStartY = touches.clientY
+        @popoverIsTouching = true
+        @popoverCancelTouch = false
+      undefined
+    @popoverEl.on 'touchmove', '.item', (e)=>
+      if @popoverIsTouching
+        touches = e.originalEvent.targetTouches[0]
+        currentY = touches.clientY
+        @popoverCancelTouch = @popoverCancelTouch || (Math.abs(@popoverStartY - currentY) > 15)
```

Comment:
@dozoisch I like the idea of assigning a variable to this number so the intent is not lost :)